src/ashtech.h

```diff
@@ -72,16 +72,16 @@ class GPSDriverAshtech : public GPSHelper
 		NME_DECODE_GOT_FIRST_CS_BYTE
 	};
 
-	struct satellite_info_s *_satellite_info;
-	struct vehicle_gps_position_s *_gps_position;
-	uint64_t _last_timestamp_time;
-	int _ashtechlog_fd;
+	struct satellite_info_s *_satellite_info {nullptr};
```

Comment:
@dagar I prefer the  `= nullptr` intialization, because it is more consistent with what we already have & are doing. What's your take? Is this actually the prefered way of doing it in C++11?